### PR TITLE
--version option added

### PR DESCRIPTION
**Pull request by sunanda690**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,6 +73,7 @@
     "dotenv": "^10.0.0",
     "flat": "^5.0.2",
     "node-vault": "^0.9.22",
+    "secretsfoundry": "^0.1.15",
```
Comment (shubham-rai-tf) [inline package.json:76]: This seems wrong. Why should secretsfoundry be a dependency for itself?

Comment (sunanda690) [reply package.json:76]: This was added by running this command `npm install secretsfoundry --save ` from readme. I will remove it

**Review by innoavator — APPROVED** [review]

File: src/index.ts [hunk 2]
```diff
@@ -4,11 +4,13 @@ import { Command } from 'commander';
 import { SecretsFoundry } from './SecretsFoundry';
 import { Loaders } from './loaders';
 import Utils, { Options } from './utils';
+import pjson = require('./../package.json');
```
Comment (innoavator) [inline src/index.ts:7]: rename it to packageJson

**Review by innoavator — COMMENTED** [review]
Just one small change suggested - then you can merge this PR

**Pull request merged**